source/common_fields.rst

```diff
@@ -0,0 +1,484 @@
+   :alt: module assets - group
+   :align: center
+   :scale: 63%
+
+- In this window, you need to add users. In **Users** tab, select those you wish to associate with the group (prefer users with a technical or higher profile)
+
+.. image:: ../image/add-users-in-group.png
+   :alt: module assets - add user in group
+   :align: center
+   :scale: 43%
+
+1. Select **user**
+2. Define it this user can be **Manage** group (add/modify memebers, name, etc.)
+3. whether or not to authorise **Delegatee** rights (can open ticket for the group)
+4. **Add** your group
+
+Once the group has been created, you can add another or select it from the drop-down list.
+
+Group in charge
+~~~~~~~~~~~~~~~
+
+A group in charge can modify the information of the computer and his elements (status, computer type, etc.)
+
+.. image:: ../image/group-in-charge.png
+   :alt: module assets - group in charge
+   :align: center
+   :scale: 51%
+
+- To create a group, click on **+ Add**
+- If it's necessary, check the box **Child entities** (this group will be visible in the sub-entity of the one you are in)
+- Enter a **Name**
+- A **Code** (optionnal)
+- Choose if this group is **Child of**
+- Select yes or no to **Recursive membership** (if enabled, members of this group will also become implicit members of its children groups)
+- You can define whether this group can be **visible in a ticket** (as a requester, observer, assigned to, task, can be notified)
+- Define if the group can be a **manager in a project**
+- Select if this group **can contain** **Items** and/or **Users**
+- Click **+ Add**
+
+.. image:: ../image/add-group-1.png
+   :alt: module assets - group in charge
+   :align: center
+   :scale: 63%
+
+.. image:: ../image/add-group-2.png
+   :alt: module assets - group in charge
+   :align: center
+   :scale: 63%
+
+- In this window, you need to add users. In **Users** tab, select those you wish to associate with the group (prefer users with a technical or higher profile)
+
+.. image:: ../image/add-users-in-group-charge.png
+   :alt: module assets - add user in group in charge
+   :align: center
+   :scale: 43%
+
+1. Select **user**
+2. Define it this user can be **Manage** group (add/modify memebers, name, etc.)
+3. whether or not to authorise **Delegatee** rights (can open ticket for the group)
+4. **Add** your group
+
+Once the group has been created, you can add another or select it from the drop-down list.
+
+.. Note:: If you cant another option with groups, see :doc:`../administration/groups`
+
+Inventory number
+~~~~~~~~~~~~~~~~
+
+**Inventory number** is information added manually. It is generally managed internally by the company.
+
+Location
+~~~~~~~~
+
+.. image:: ../image/location.png
+   :alt: module assets - location
+   :align: center
+   :scale: 50%
+
+You can add location manually or dynamically.
+
+.. warning:: Depending on your needs, pay attention to the entity you are in and its recursivity, so that it is visible or not to other entities.

```

Comment:
Not only for locations. I guess this should be a more generic warning,

Reply:
Done